### DISCUSSION TITLE
How to set width of the image programmatically on upload

### DISCUSSION THREAD

**Discussion by jay-softsages** [GrapesJS/grapesjs#4879] [Q&A] [answered]
I am using Grapesjs MJML plugin and I need to set the width of an image when it is uploaded and used. By default when we upload the image its width in the decoration tab is "auto" or empty but I need to set it to "100px" by default.

How can I achieve this? please help!

**Comment by artf** [comment, marked as answer]
You can [update the block](https://grapesjs.com/docs/modules/Blocks.html#programmatic-usage) used for the image with your property of choice.

**Reply by jay-softsages** [reply]
Thanks for the reply!

As I am using the MJML plugin I tried to get the image block by - 

```javascript
const bm = editor.Blocks;
const imageBlock = bm.get("mj-image");
imageBlock.set({});
```

I tried reading the docs but I am not aware of how I can set the width property for the element.

**Reply by artf** [reply]
`imageBlock.set({ content: '<mj-image src="DEFAULT_SRC" width="300px"/>' })`
